### ISSUE TITLE
[Feature request] Main tree in subfiles

### ISSUE THREAD

**Issue by GeoffreyV** [raphael-cch/latex-tree#16]
First, thanks for this very useful package ! :) 
Would be great to show the root tree instead of the local one in a multi-file latex project.
Related to #13 and #3

**Comment by melsophos** [comment]
I completely agree with this suggestion. At least one could have an option to choose what to show.
